src/m365/spo/commands/listitem/listitem-attachment-remove.ts

```diff
@@ -0,0 +1,165 @@
+import { Cli } from '../../../../cli/Cli.js';
+import { Logger } from '../../../../cli/Logger.js';
+import GlobalOptions from '../../../../GlobalOptions.js';
+import request, { CliRequestOptions } from '../../../../request.js';
+import { formatting } from '../../../../utils/formatting.js';
+import { urlUtil } from '../../../../utils/urlUtil.js';
+import { validation } from '../../../../utils/validation.js';
+import SpoCommand from '../../../base/SpoCommand.js';
+import commands from '../../commands.js';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  webUrl: string;
+  listId?: string;
+  listTitle?: string;
+  listUrl?: string;
+  listItemId: string;
+  fileName: string;
+  confirm?: boolean;
```

Comment:
Let's change confirm for force. Ref: https://github.com/pnp/cli-microsoft365/issues/4551